### ISSUE TITLE
old version of htmlbars set as dependacy

### ISSUE THREAD

**Issue by evoactivity** [khorus/ember-accordion#6]
This addon specifies 0.7.9 while the latest version is 1.1.1, that seems quite out of date and it also causes a deprecation warning

```bash
DEPRECATION: Overriding init without calling this._super is deprecated. 
Please call `this._super.init && this._super.init.apply(this, arguments);` 
addon: `ember-cli-htmlbars`
```

Does this dependancy even need to be defined by the addon since it will be installed by default by ember?

**Comment by chbonser** [comment]
Hi @evoactivity I'll take a look at this on Friday.  Thanks for the feedback.

**Comment by chbonser** [comment]
This is resolved in v0.4.0.  See updated readme for upgrade instructions.
